submarine-cloud/submarine-operator.md

```diff
@@ -0,0 +1,155 @@
+# Submarine Operator Development Instructions
+
+See also official sample：https://github.com/kubernetes/sample-controller
+
+# Implementation steps
+
+## 1. Submit the CRD template and its instantiation object to the k8s cluster, so that k8s can recognize
+```
+1 Official document CRD 
+https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition
+
+2 Log in to a machine that can execute kubectl commands and create a student.yaml file
+[root@localhost student]# kubectl apply -f student.yaml
```

Comment:
Looks like this submarine-operator.md needs changes according to submarine operator, Right?